lib/pbench/agent/tool_meister.py

```diff
@@ -1728,22 +1727,31 @@ def get_logger(PROG: str, daemon: bool = False) -> logging.Logger:
     return logger
 
 
+class Arguments(NamedTuple):
+    host: str
+    port: int
+    key: str
+    daemonize: bool
+    level: str
```

Comment:
This is the same definition as `tool_data_sync.Arguments`...should it be shared?